assets/src/components/SelectCard.js

```diff
@@ -1,11 +1,13 @@
 /* global fetch */
 import React, { useState } from 'react'
 import { withStyles } from '@material-ui/core/styles'
-import { Card, CardActionArea, CardActions, CardContent, CardMedia, CircularProgress, Divider, Fab, IconButton, Snackbar, Typography } from '@material-ui/core'
+import { Card, CardActionArea, CardActions, CardContent, CardMedia, CircularProgress, Divider, Fab, IconButton, Link as MUILink, Snackbar, Tooltip, Typography } from '@material-ui/core'
 import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank'
 import CheckBoxIcon from '@material-ui/icons/CheckBox'
 import CloseIcon from '@material-ui/icons/Close'
 import { Link } from 'react-router-dom'
+import InfoIcon from '@material-ui/icons/Info'
```

Comment:
The material-ui documentation says individual imports is 'safer' whereas one line yield the 'best developer experience' however you had to take care to [minimize your bundle size](https://material-ui.com/components/icons/#usage) if you do it that way.  I'm not sure if that is something we're already doing appropriately?

Reply:
Good point, didn't think about that. I wasn't necessarily saying that it should be all on one line. At the least, the icon imports could be one after another.

Reply:
Seems like we're probably safe to use "tree-shaking", per the ES6 modules and `webpack` versions limitations listed here: https://material-ui.com/guides/minimizing-bundle-size/#when-and-how-to-use-tree-shaking

@pushyamig, can you confirm?

Reply:
It's different for icons I guess. We have this babel import (`babel-plugin-named-asset-import`), not sure if that does it.

Reply:
I'll admit I'm ignorant about both `webpack` and `babel`, so thanks for raising this issue.

Reply:
I am too, I didn't want to make a change that might have a negative impact on the  bundles size.

Reply:
We are on Webpack 4 i see the tree-shaking is supported that particular version. For prod webpack build it does minimize the bundle size. bundle can be further compressed as I am doing it [here](https://github.com/pushyamig/my-learning-analytics/blob/i987_optimize_webpack/package.json#L137) and [here](https://github.com/pushyamig/my-learning-analytics/blob/i987_optimize_webpack/webpack.config.js#L38) 

but we can even break down the bundle size using webpack code-[splitting](https://webpack.js.org/guides/code-splitting/) instead of one single js file we can have multiple as canvas does 

![Screen Shot 2020-08-14 at 12 10 43 PM](https://user-images.githubusercontent.com/8579775/90269800-37d4b880-de27-11ea-9105-599820d45bf5.png)

Whatever chris is doing is fine, we might need to look at code-splitting as a future issue

Reply:
> It's different for icons I guess. We have this babel import (babel-plugin-named-asset-import), not sure if that does it.

Probably it is doing it when making it as big bundle, since in the `webpack.config.js`. 
```
rules: [
      {
        test: /\.(js|mjs|jsx|ts|tsx)$/,
        loader: 'babel-loader'
```

I really don't know details of it. when making the bundle it just add a simple import in the js file instead of multiple reference based on the how many file it reference. I am learning this as well